src/components/credentials/credentials.js

```diff
@@ -125,7 +126,7 @@ class Credentials extends React.Component {
           rows={viewOptions.selectedItems.length}
           value={credentialsList}
         />
-      </Grid.Col>
+      </List>
```

Comment:
required
- lets remove the `List` component entirely since it's wrapping a form `textarea`. Plus, it doesn't appear to cause any visual difference by removing it

Reply:
Good call - updated!